.github/actions/upload-release-and-artifact/action.yml

```
@@ -0,0 +1,49 @@
+name: "Upload Release Aassets and Artifacts"
+description: |
+  releaseのassetとartifactにアップロードする。
+  release・artifactにアップロードするかはそれぞれ制御できる。
+
+inputs:
+  files:
+    description: "アップロードするファイルのパス。ワイルドカードも使える。複数指定する場合は改行で区切る。"
+    type: string
+    required: true
+  upload_release:
+    description: "releaseにアップロードするか。"
+    type: boolean
+    default: false
+  upload_artifact:
+    description: "artifactにアップロードするか。"
+    type: boolean
+    default: false
+  prerelease:
+    description: "pre-releaseかどうか。"
+    type: boolean
+    default: false
+  tag_name:
+    description: "タグ名。"
+    type: string
+  target_commitish:
+    description: "タグを付けるコミットID。"
+    type: string
+  artifact_name:
+    description: "artifactの名前"
```

```suggestion
    description: "artifact名。"
```
めちゃめちゃどうでもいいですが他の`description`に合わせてみました(`。`で終わるのと、`タグ名`に合わせた)